bash-preexec.sh

```diff
@@ -106,7 +106,7 @@ __bp_precmd_invoke_cmd() {
         # Test existence of functions with: declare -[Ff]
         if type -t "$precmd_function" 1>/dev/null; then
             __bp_set_ret_value "$__bp_last_ret_value" "$__bp_last_argument_prev_command"
-            $precmd_function
+            "$precmd_function"
```

Comment:
It would be good to leave a comment in the file somewhere noting why we need to quote function names.

Reply:
https://github.com/rcaloras/bash-preexec/commit/331fa90ba5575369f27afbd17ca1afb23e0ce4c2